aiida/orm/data/structure.py

```diff
@@ -1160,17 +1148,34 @@ def append_site(self, site):
                 "The StructureData object cannot be modified, "
                 "it has already been stored")
 
-        new_site = Site(site=site)  # So we make a copy
+        #new_site = Site(site=site)  # So we make a copy
```

Comment:
I was having a lot of problems with performance importing structures through ase and into AiiDA.  Most of problem stems from there being multiple redundant (deep)copies.  So, for example on line [1160](https://github.com/muhrin/aiida_core/blob/call_old_wf_from_new/aiida/orm/data/structure.py#L1160)  we call get_raw which returns a new dictionary where internally position gets deep copied and kind is immutable.  Then, in _set_attr the whole thing is deep copied _again_, so this way I get rid of at least one copy.